Source/Car_BC.hpp

```diff
@@ -191,6 +192,26 @@ class Car_6C
 
 EXPORT_VAR extern Car_6C* gCar_6C_677930;
 
+class BitSet32_Car_BC {
```

Comment:
might be worth moving this to a separate file when this is used in other classes as well (it probably is, but everything is most likely inlined).